app/src/main/res/drawable/ic_desktop.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+    <path android:fillColor="?primaryText"
+        android:fillType="evenOdd"
```

Comment:
Android has some issues filling with evenOdd while using attributes. Can you change this to be `nonzero`?

Reply:
Mmm we usually can't just swap `evenOdd` for `nonZero`. First try to remove it altogether and see if the icon still looks right, otherwise we'll have to get a new asset from the designers.

Reply:
Neither of the solutions work, the icon is changed.  We already have other 4 icons in Fenix and 2 in A-C that use this attribute. There is a desktop icon in A-C that does not use that attribute and it looks the same. 

```
<vector xmlns:android="http://schemas.android.com/apk/res/android"
        android:width="24dp"
        android:height="24dp"
        android:viewportWidth="24"
        android:viewportHeight="24">
  <path android:fillColor="@color/mozac_ui_icons_fill"
        android:pathData="M4,6a1,1 0,0 1,1 -1h14a1,1 0,0 1,1 1v11h1.5a0.5,0.5 0,0 1,0.5 0.5v1a0.5,0.5 0,0 1,-0.5 0.5h-19a0.5,0.5 0,0 1,-0.5 -0.5v-1a0.5,0.5 0,0 1,0.5 -0.5L4,17L4,6zM10,17v1h4v-1h-4zM6,7v9h12L18,7L6,7z" />
</vector>
```

Can we use this? Changing the fill colour, of course.

Reply:
@mcarare seems reasonable to use that icon, yeah, just change the fill color to `?primaryText`